frontend/test/acceptance/JoinSupporterSpec.scala

```diff
@@ -30,14 +30,15 @@ class JoinSupporterSpec extends FeatureSpec with Browser
   }
 
   feature("Become a Supporter in UK") {
+
     scenario("User joins as Supporter by clicking 'Become a Supporter' button on Membership homepage", Acceptance) {
```

Comment:
I would rename this scenario as: "User joins as Supporter using Stripe" so it is consistent to the new paypal scenario.